platform/linuxbsd/display_server_x11.cpp

```diff
@@ -3352,8 +3356,9 @@ DisplayServerX11::WindowID DisplayServerX11::_create_window(WindowMode p_mode, u
 		XSync(x11_display, False);
 		XGetWindowAttributes(x11_display, wd.x11_window, &xwa);
 
-		wd.position.x = xwa.x;
-		wd.position.y = xwa.y;
+		wd.position.x = p_rect.position.x;
+		wd.position.y = p_rect.position.y;
```

Comment:
It this correct with tiling window managers? Created window may have different size from the requested one.

Reply:
Currently that's the only way I found it to work (at least with non tiled window managers). I tried this with i3 running together with KDE and seems to make popups like a new window just next to the editor. The size is not ok but the user can work with it I think.

However, I didn't remember to resize or move the popup (it is possible with programs like Spectacle which I used to take the photo, to move them around even if it's a tiled window manager) but it may use a special configuration (I'm not sure).

![image](https://user-images.githubusercontent.com/45542433/79116783-a3c7e700-7d89-11ea-9ac0-7468da522bf7.png)

Reply:
Yes I have found that windows can be set to float in tiled WMs , so it shouldn't be a problem unless someone raises a problem about it :+1: